virttest/utils_misc.py

```diff
@@ -2370,6 +2370,59 @@ def get_qemu_best_cpu_model(params):
     return params.get("default_cpu_model", None)
 
 
+def get_qemu_version(params):
+    """
+    Get the qemu-kvm(-rhev) version info
+    """
+    major_version = 0
+    minor_version = 0
+    update_version = 0
+    is_rhev = False
+    version = {}
+    regex = r'\s*[Ee]mulator [Vv]ersion\s*(\d+)\.(\d+)\.(\d+)'
+
+    qemu_binary = get_qemu_binary(params)
+    version_raw = commands.getoutput("%s -version" % qemu_binary).splitlines()
+    for line in version_raw:
+        search_result = re.search(regex, line)
+        if bool(search_result):
+            major_version = int(search_result.group(1))
+            minor_version = int(search_result.group(2))
+            update_version = int(search_result.group(3))
+        if "rhev" in str(line).lower():
+            is_rhev = True
+    version['major'] = major_version
+    version['minor'] = minor_version
+    version['update'] = update_version
+    version['is_rhev'] = is_rhev
+    return version
+
+
+def compare_qemu_version(major, minor, update, is_rhev=True, params={}):
+    """
+    Check if local install qemu versions is newer than provided version.
+    :param major: The major version to be compared.
+    :param minor: The minor version to be compared.
+    :param update: The update version to be compared.
+    :param is_rhev: If the qemu is a rhev version.
+    :param params: Other params.
+    :return: True means local installed version is equal or newer than the
+    version provided, otherwise False will be returned.
+    """
+    installed_version = get_qemu_version(params)
+    if bool(is_rhev) != bool(installed_version['is_rhev']):
```

Comment:
Is there any reason that need to use bool here?

Reply:
Just thought this could be more readable, but actually this will cause some potential problem since
bool("abc") == bool("def") will return True.
So fixed.